### PR TITLE
Fix issue 771 by implementing S conversion specifier for printf-related functions

### PR DESCRIPTION
**Pull request by GabrielRavier**
(see also https://github.com/jart/cosmopolitan/issues/771)

### CODE REVIEW

File: test/libc/fmt/fmt_test.c [hunk 1]
```diff
@@ -64,6 +64,10 @@ TEST(fmt, d) {
   EXPECT_STREQ("-2147483648", _gc(xasprintf("% d", INT_MIN)));
 }
 
+TEST(fmt, s) {
+  EXPECT_STREQ("Wide character output test", _gc(xasprintf("%S", L"Wide character output test")));
```
Comment (G4Vi) [inline test/libc/fmt/fmt_test.c:68]: This line looks long, please see the style guide https://github.com/jart/cosmopolitan/blob/master/CONTRIBUTING.md#style-guide In your own test files you can `/* clang-format off */` and use your own styling.

**Review by G4Vi — APPROVED** [review]
LGTM. thank you!

**Comment by G4Vi** [comment]
I fixed the styling in the test file, see the note in the review.

File: test/libc/fmt/fmt_test.c (hunk 1 above)
Comment (GabrielRavier) [reply test/libc/fmt/fmt_test.c]: Ah, sorry, forgot to run `clang-format` on it. Thanks for fixing that!

**Pull request merged**